ex1-tema2.js

```diff
@@ -7,4 +7,22 @@ function even_numbers (n){
 }
 even_numbers(10);
 
+//Tipăriți toate numerele impare  de la 0 la 10
 
+//Method 1
+function even_numbers (n){
```

Comment:
(opt) Chestii de finete:

```suggestion
function even_numbers(n) {
```